internal/kafka/internal/migrations/migrations.go

```diff
@@ -60,6 +60,7 @@ var migrations = []*gormigrate.Migration{
 	addKafkaInstanceTypeColumn(),
 	addKafkaCanaryServiceAccountColumns(),
 	addKafkaNamespaceColumn(),
+	addAdminApiServerURL(),
```

Comment:
let's respect the ordering in respect to what already exists when adding a new migration. 
So let's move this to the end of the array

Reply:
Of course, thanks @machi1990 
Thanks for the other feedback and clarification too :)